### PR TITLE
Update CoinChange.java

### PR DESCRIPTION
**Pull request by roshinjojuc**

### CODE REVIEW

File: dynamicProgramming/CoinChange.java [hunk 1]
```diff
@@ -35,9 +35,6 @@
  * The base case is that for making amount zero, we need Zero coins.
  * 
  * We can define a dp array and initialize this as the base case. For all other index, lets first assume that the amount cannot be given with the coins and initialize them as the maximum number, i have used 10001 as the amount can be anthing in range 0 to 10000.
- * We will try to solve this using tabulation using a dp array and a max variable to store the max length among all increasing sequences.
```
Comment (athulsukumaran) [inline dynamicProgramming/CoinChange.java:38]: Is this intentional?

Comment (athulsukumaran) [reply dynamicProgramming/CoinChange.java:38]: @roshinjojuc Is this intentional?

Comment (roshinjojuc) [reply dynamicProgramming/CoinChange.java:38]: no this must be a copy edit mistake, will remove it now

**Comment by roshinjojuc** [comment]
@athulsukumaran The lines are removed by you it seems, do approve this now.

**Comment by roshinjojuc** [comment]
@athulsukumaran The lines are removed by you it seems, do approve this now.

**Comment by athulsukumaran** [comment]
We don't need this change

**Comment by roshinjojuc** [comment]
@athulsukumaran its not merged, reopening again

**Review by athulsukumaran — APPROVED** [review]

**Pull request merged**